### DISCUSSION TITLE
Integration with Steam

### DISCUSSION THREAD

**Discussion by hadesfury** [3dproger/AxelChat#489] [Ideas]
Hello, your software is really awesome, thank you for making it. 

I wonder how hard do you think it is to integrate with the steam page chat.

Best regards.

**Comment by 3dproger** [comment]
Good idea, I'll try to do it in the future

**Comment by 3dproger** [comment]
Do you mean integrate with the live stream chat on Steam? Or do you mean something else?

**Reply by hadesfury** [reply]
It would be awesome if the Steam chat would be merged with the others, a new integration like that youtube or twitch one 

sorry if I was not clear enough

**Reply by 3dproger** [reply]
Okay, I'll try to do that

**Comment by 3dproger** [comment]
Hi! I finally made it Steam support! You can download: https://github.com/3dproger/AxelChat/releases
I'll close this discussion. If you have any questions, write in a new one
